saleor/warehouse/models.py

```diff
@@ -0,0 +1,62 @@
+import uuid
+from typing import Set
+
+from django.db import models
+from django.utils.translation import pgettext_lazy
+
+from ..account.models import Address
+from ..shipping.models import ShippingZone
+
+
+class WarehouseQueryset(models.QuerySet):
+    def prefetch_data(self):
+        return self.select_related("address").prefetch_related("shipping_zones")
+
+
+class Warehouse(models.Model):
+    id = models.UUIDField(default=uuid.uuid4, primary_key=True)
+    name = models.CharField(
```

Comment:
Will the name be presented?  If so, do we need translations for its name?

Reply:
The name will be presented only in the dashboard. If there is any need for translating this - I do not know anything about this.